cli/src/command_utils.rs

```diff
@@ -0,0 +1,91 @@
+/*
+	Copyright 2021 Integritee AG and Supercomputing Systems AG
+
+	Licensed under the Apache License, Version 2.0 (the "License");
+	you may not use this file except in compliance with the License.
+	You may obtain a copy of the License at
+
+		http://www.apache.org/licenses/LICENSE-2.0
+
+	Unless required by applicable law or agreed to in writing, software
+	distributed under the License is distributed on an "AS IS" BASIS,
+	WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+	See the License for the specific language governing permissions and
+	limitations under the License.
+
+*/
+
+use crate::Cli;
+use sgx_crypto_helper::rsa3072::Rsa3072PubKey;
+
+use sp_application_crypto::sr25519;
+use std::path::PathBuf;
+
+use codec::Encode;
+use log::*;
+use my_node_runtime::{AccountId, Signature};
+use sp_core::{crypto::Ss58Codec, Pair};
+use sp_runtime::traits::{IdentifyAccount, Verify};
+use substrate_api_client::{rpc::WsRpcClient, Api};
+
+use itc_rpc_client::direct_client::{DirectApi, DirectClient as DirectWorkerApi};
+use substrate_client_keystore::LocalKeystore;
```

Comment:
Just a minor thing: Since we already move this code, we should also group all `use` together and let `cargo fmt` define the ordering.

Reply:
ok!